composer.json

```diff
@@ -17,11 +17,11 @@
         "github-domains": ["github.com"]
     },
     "require": {
-        "php": ">=5.3.2"
+        "php": ">=5.3.2",
+        "symfony/framework-bundle": ">=3.0",
```

Comment:
Symfony\Component\Security\Core\SecurityContext has been deprecated in Symfony 2.6, so the version-requirement here should be ">=2.6 || >=3.0" for the two symfony bundles. Don't you think?